client/flags.go

```diff
@@ -116,7 +118,9 @@ func RegisterRestServerFlags(cmd *cobra.Command) *cobra.Command {
 	cmd = GetCommands(cmd)[0]
 	cmd.Flags().String(FlagListenAddr, "tcp://localhost:1317", "The address for the server to listen on")
 	cmd.Flags().String(FlagCORS, "", "Set the domains that can make CORS requests (* for all)")
-	cmd.Flags().Int(FlagMaxOpenConnections, 1000, "The number of maximum open connections")
+	cmd.Flags().Uint(FlagMaxOpenConnections, 1000, "The number of maximum open connections")
+	cmd.Flags().Uint(FlagRPCReadTimeout, 10, "The RPC read timeout (in seconds)")
+	cmd.Flags().Uint(FlagRPCWriteTimeout, 10, "The RPC write timeout (in seconds)")
```

Comment:
Is this the timeout in order to receive conformation that your write request was successful? Like what is a rest write?

Reply:
@rigelrozanski yeah, the write timeout, from the REST client pov, is mainly for broadcasting.